src/components/Menu/Menu.scss

```diff
@@ -0,0 +1,202 @@
+@media screen and (max-width: 1343px) {
+  .menu-bars {
+    position: fixed;
+    left: calc(50% - 134px);
+    bottom: 8px;
+  }
+  .menu-bars--isAdmin {
+    left: calc(50% - 168px);
+  }
+  .menu-bars--user {
+    @media (prefers-color-scheme: light) {
+      .menu-bars__switch {
+        background-color: #FFFFFF;
+        box-shadow: 0 16px 32px 0 rgba(0, 87, 255, 0.24);
+      }
+      .menu-bars__switch > .switch__icon {
+        color: #ff0069;
+      }
+    }
+    @media (prefers-color-scheme: dark) {
+      .menu-bars__switch {
+        background-color: #ff0069;
+        box-shadow: 0 6px 9px 0 rgba(0, 0, 0, 0.2);
+      }
+      .menu-bars__switch > .switch__icon {
+        color: #FFFFFF;
+      }
+    }
+  }
+  .menu-bars--user > .user-menu {
+    visibility: visible;
+    opacity: 1;
+    bottom: 8px;
+  }
+  .menu-bars--user > .admin-menu {
+    visibility: hidden;
+    opacity: 0;
+    bottom: 72px;
+  }
+  .menu-bars--user > .menu-bars__switch > .switch__icon--add {
+    visibility: hidden;
+    opacity: 0;
+    transform: rotate(360deg);
+  }
+  
+  .menu-bars--admin {
+    @media (prefers-color-scheme: light) {
+      .menu-bars__switch {
+        background-color: #FFFFFF;
+        box-shadow: 0 16px 32px 0 rgba(0, 87, 255, 0.24);
+      }
+      .menu-bars__switch > .switch__icon {
+        color: #0057ff;
+      }
+    }
+    @media (prefers-color-scheme: dark) {
+      .menu-bars__switch {
+        background-color: #0057ff;
+        box-shadow: 0 6px 9px 0 rgba(0, 0, 0, 0.2);
+      }
+      .menu-bars__switch > .switch__icon {
+        color: #FFFFFF;
+      }
+    }
+  }
+  .menu-bars--admin > .admin-menu {
+    visibility: visible;
+    opacity: 1;
+    bottom: 8px;
+  }
+  .menu-bars--admin > .user-menu {
+    visibility: hidden;
+    opacity: 0;
+    bottom: -56px;
+  }
+  .menu-bars--admin > .menu-bars__switch > .switch__icon--settings {
+    visibility: hidden;
+    opacity: 0;
+    transform: rotate(-360deg);
+  }
+  .menu-bars__switch {
+    height: 64px;
+    width: 64px;
+    padding: 0;
+    border: 0;
+    border-radius: 100%;
+    transition: background-color 0.25s ease-in-out;
+    margin-left: 272px;
+    position: relative;
+  }
+  .menu-bars__switch:focus {
+    outline: none;
+  }
+  
+  .switch__icon {
+    position: absolute;
+    top: calc(50% - 18px);
+    left: calc(50% - 18px);
+    transform: rotate(0deg);
+    transition: opacity 0.5s, visibility 0.5s, transform 1s;
+  }
+}
+
+@media screen and (min-width: 1344px) {
+  .menu-bars__switch {
+    display: none;
+  }
+}
+
+.user-menu {
+  --primaryColor: #0057ff;
+  --secondaryColor: #253f71;
+}
+
+.admin-menu {
+  --primaryColor: #ff0069;
+  --secondaryColor: #7c123e;
+}
+
+@media screen and (max-width: 1343px) {
+  .menu {
+    border-radius: 34.5px;
```

Comment:
Is there such a thing as decimal px sizes? I think it might be relevant if you start zooming your browser view but it seems fishy somehow

Reply:
I changed it to 35px because I hasn't made a difference fully zoomed in